src/Apps/Order/Dialogs.tsx

```diff
@@ -56,6 +56,47 @@ export class DialogContainer extends Container<DialogState> {
     })
   }
 
+  showAcceptDialog = ({
```

Comment:
Back when I originally wrote this it was named `accept` after dual-option 'accept/cancel' dialogs, rather than the `/accept` page but yeah it's confusing and I like your name better so :+1:

Reply:
Actually how do you feel about `showConfirmDialog`? I'm thinking of `window.confirm()`

Reply:
Oh! Yeah that sounds 👍